src/Grok.Net/Grok.cs

```diff
@@ -86,6 +86,41 @@ public GrokResult Parse(string text)
             return new GrokResult(grokItems);
         }
 
+        public Dictionary<string, List<object>> AsDictionary(string text)
```

Comment:
AFAIK this violates dotnet convention of `AsX()` functions not allocating though I can't find a definitive source on this right now :/ Extension method of `GrokResult.ToDictionary()` might be more correct?

Reply:
good point. thx